dpnp/backend/tests/test_utils_iterator.cpp

```diff
@@ -429,6 +429,40 @@ TEST_P(IteratorReduction, sycl_reduce_axis)
     }
 }
 
+TEST(TestUtilsIterator, sycl_get_first)
+{
+    using data_type = double;
```

Comment:
no needed.

Reply:
I'm not going to merge the test to master, because the test is overlapped by other existing test. It's just a simple issue reproducer.

Reply:
you can write it down for all comments, or just leave the link.

Reply:
it is better rename pr to : `REPRODUCER: ...`